aws/resource_aws_launch_template_test.go

```diff
@@ -122,6 +122,7 @@ func TestAccAWSLaunchTemplate_data(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "block_device_mappings.#", "1"),
 					resource.TestCheckResourceAttr(resName, "credit_specification.#", "1"),
 					resource.TestCheckResourceAttrSet(resName, "disable_api_termination"),
+					resource.TestCheckResourceAttrSet(resName, "ebs_optimized"),
```

Comment:
Rather than just testing the attribute being set (to anything) we should ensure its value is correct:

```go
resource.TestCheckResourceAttr(resName, "ebs_optimized", "false"),
```

We should also ensure at least one test whose configuration doesn't include `ebs_optimized` has the correct value as well:

```
resource.TestCheckResourceAttr(resName, "ebs_optimized", ""),
```